torchci/pages/api/classifier/rules.ts

```diff
@@ -89,6 +89,11 @@ export default async function handler(
         pattern: r`^FAILED: Build did NOT complete successfully`,
         priority: 999,
       },
+      {
+        name: "Python flaky unittest",
+        pattern: r`^\s*(test.*) (fail|error|succeed)?ed - num_retries_left:`,
```

Comment:
Here, I'm intending the capture groups to be (test.*) and (fail|error|succeed).

I also set the priority to be the same as a unittest failure but I'm thinking it should be lower priority as a failure. Would 999 be a better value?

Reply:
Couldn't really parse your question; the classifier takes the higher priority classification so you can set it however you want to get your desired behavior.